### ISSUE TITLE
ga-lite public API

### ISSUE THREAD

**Issue by jehna** [jehna/ga-lite#7]
So this project can be easily extended, the library should expose a public API that a user can call.

Similarly like the [official analytics.js](https://developers.google.com/analytics/devguides/collection/analyticsjs/events) exposes the `ga()` function, ga-lite should have some kind of public API that can be called when an event has fired.

**Comment by Enalmada** [comment]
On my website I am changing pages with pjax and manually calling this when content/unl has changed:
```
ga('set', 'location', window.location.href);
ga('send', 'pageview');
```
Can I accomplish this equivalent with "ga-lite" or is the public api necessary?

**Comment by jehna** [comment]
@Enalmada no, `ga-lite` does not support manual events yet. The version 2.0 would be needed to do this.

**Comment by jehna** [comment]
Implemented in `version-2.0` branch
